### DISCUSSION TITLE
A To Do section

### DISCUSSION THREAD

**Discussion by anujbarochia** [dailydotdev/daily#915] [Feature Requests]
Please add a todo section in the left part of the home screen of daily.dev It will be easy for me to navigate between my daily tasks and the reading articles on 1 app Daily.dev Because i might just sometimes need todo more than reading articles, but i want to do both at one place(i.e Daily.dev). So it is a request i think you might fulfill.

**Comment by upieez** [comment]
I think that's a great idea! Maybe to take this a step further, would it be possible to integrate github issues to my todo list? so that at a glance, I can see what are some pending issues that I have.

**Comment by idoshamun** [comment]
Hi @anujbarochia and @upieez ,

Thanks a lot for the suggestion :)

**Comment by gwydion67** [comment]
it wouldn't be an exaggeration to say over 70% of my screen time goes to my browser and i think this is the normal for most of the students like me. having a sleek and short ToDo/note on the new tab can give a quick overlook to what i have to do for a short time frame without wasting time over opening some app and do the same.
